usaspending_api/common/validator/helpers.py

```diff
@@ -179,7 +179,9 @@ def validate_object(rule):
     for key, value in rule["object_keys"].items():
         if key not in provided_object:
             if "optional" in value and value["optional"] is False:
-                raise UnprocessableEntityException("Required object fields: {}".format(rule["object_keys"].keys()))
```

Comment:
Using f-string might be short enough for single line

Reply:
Good call, will change this.